src/control/server/storage/bdev/backend_test.go

```diff
@@ -917,28 +922,18 @@ func TestBackend_hugePageWalkFn(t *testing.T) {
 
 func TestBackend_Prepare(t *testing.T) {
 	const (
-		testNrHugePages       = 8192
+		testNrHugepages       = 8192
 		nonexistentTargetUser = "nonexistentTargetUser"
 		username              = "bob"
 	)
 
-	mockAddrList := func(t *testing.T, idxs ...int) *hardware.PCIAddressSet {
-		var addrs hardware.PCIAddressSet
-
-		for _, idx := range idxs {
-			if err := addrs.AddStrings(common.MockPCIAddr(int32(idx))); err != nil {
-				t.Fatal(err)
-			}
-		}
-
-		return &addrs
-	}
-
-	testPCIAllowList := mockAddrList(t, 1, 2, 3).String()
-	testPCIBlockList := mockAddrList(t, 4, 3).String()
+	//mockAddrList(t, 1,2,3).String() := mockAddrList(t, 1, 2, 3).String()
+	//testPCIBlockList := mockAddrList(t, 4, 3).String()
```

Comment:
Some code commented here - does it need to be re-enabled?

Reply:
no just cruft, I will remove

Reply:
done